app/src/main/kotlin/com/wire/android/ui/common/topappbar/EstablishedCallState.kt

```diff
@@ -0,0 +1,10 @@
+package com.wire.android.ui.common.topappbar
+
+import com.wire.kalium.logic.data.id.ConversationId
+
+data class EstablishedCallState(
+    val conversationId: ConversationId? = null,
+    val isCallHappening: Boolean = false,
```

Comment:
I'm wondering is this `isCallHappening` the same as an `OnGoingCall` ? Just if it makes sense to change the naming 🤔  but can't actually tell.

Reply:
this "extra" variable is just so we can properly listen to the current screen shown where this component is currently being used, otherwise probably the status bar would show in other screens where we still don't want it. As there is another ticket to handle other screens.

Do you think changing it to `isOngoingCallHappening` more suitable? I haven't thought on a proper name because the name of the data class `_Establised_CallState` should already impose what type of call status we are working with

Reply:
I think is ok like that, don't need for change, was just a question regarding if conceptually was the same eheh 🙌